config.yml

```diff
@@ -19,16 +19,16 @@ observer:
   enabled: false
   # TODO : Add description
   auth: test
-  # Redis Subscription Database
-  redis: redis://redis:6379
```

Comment:
it is duplicate, now we have `storage.redis`

Reply:
Why touching this code in this PR?

Reply:
This requires changes in deployment pipeline.

Reply:
I going remove and change in another PR, we use `storage.redis` now